### PR TITLE
Add export for totpMultifactorGenerator for react native bundle

### PR DESCRIPTION
**Pull request by bhparijat**
PR for fixing github issue https://github.com/firebase/firebase-js-sdk/issues/7701 to provide support for Totp for react native apps.

### CODE REVIEW

File: packages/auth/index.rn.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export {
 
 // MFA
 export { PhoneMultiFactorGenerator } from './src/platform_browser/mfa/assertions/phone';
+export {TotpMultiFactorGenerator} from './src/mfa/assertions/totp';
```
Comment (prameshj) [inline packages/auth/index.rn.ts:50]: We should also include TotpSecret

**Comment by prameshj** [comment]
Also please add a changeset using `yarn changeset add`. Thanks!

**Review by prameshj — APPROVED** [review]

**Comment by prameshj** [comment]
@hsubox76 @DellaBitta requesting approval on this one.. Thanks!

File: packages/auth/index.rn.ts (hunk 1 above)
Comment (bhparijat) [reply packages/auth/index.rn.ts]: Done

**Review by renkelvin — APPROVED** [review]

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**